db-engine/src/main/java/com/splicemachine/db/iapi/sql/conn/LanguageConnectionContext.java

```diff
@@ -282,6 +282,15 @@ void removeActivation(Activation a)
      */
     boolean isVisibleToCurrentSession(TableDescriptor td) throws StandardException;
 
+    /**
+     * Return the creating session ID of a local temporary table
+     *
+     * @param td TableDescriptor of a local temporary table
+     * @return The creating session ID of this local temporary table
+     * @throws StandardException
+     */
+    java.util.UUID getLocalTempTableSessionID(TableDescriptor td) throws StandardException;
```

Comment:
I don't think this method really belongs here, but I don't have a better suggestion.